dependency/src/main/java/de/dagere/peass/execution/gradle/AnboxTestExecutor.java

```diff
@@ -52,7 +71,7 @@ private Process buildGradleProcess(final File moduleFolder, final File logFile,
       String[] anboxOriginals = new String[] { "adb", "shell", "am", "instrument", "-w", "-e", "class" };
```

Comment:
Shouldn't `fetchAdbCall()` be called here as well, instead just using `adb`?

Reply:
It makes sense. I've changed that as well.